plugins/json-logger/webpack.config.js

```diff
@@ -0,0 +1,44 @@
+const path = require("path");
+const webpack = require("webpack");
+const CopyWebpackPlugin = require("copy-webpack-plugin");
+const nodeExternals = require("webpack-node-externals");
+
+/** @type {import("webpack").Configuration} */
+module.exports = {
+  mode: "production",
+  target: "node",
+  entry: "./src/index.ts",
+  plugins: [
+    new webpack.SourceMapDevToolPlugin({
+      filename: "[name].js.map",
+    }),
+    new CopyWebpackPlugin({
+      patterns: [
+        { from: "src/static", to: "static", noErrorOnMissing: true },
+        { from: "src/templates", to: "templates", noErrorOnMissing: true },
+      ],
+    }),
+  ],
+  module: {
+    rules: [
+      {
+        test: /\.tsx?$/,
+        use: "ts-loader",
+        exclude: /node_modules/,
+      },
+    ],
+  },
+  resolve: {
+    extensions: [".ts", ".js", ".json"],
+  },
+  optimization: {
+    minimize: false,
+  },
+  output: {
+    filename: "index.js",
+    path: path.resolve(__dirname, "dist"),
+    libraryTarget: "commonjs2",
+    clean: true,
+  },
+  externals: [nodeExternals()],
```

Comment:
Remove this line as we need all plugin to bundle all their dependencies
more info: https://github.com/amplication/amplication/issues/7150